doc/api/child_process.md

```diff
@@ -299,6 +299,9 @@ output on this fd is expected to be line delimited JSON objects.
 *Note: Unlike the fork(2) POSIX system call, `child_process.fork()` does
 not clone the current process.*
 
+*Note*: Support for non-Array values of the `.stdio` option was added in
```

Comment:
I don't know if we want to start adding this kind of information in this way or not. I think it's probably a separate discussion to be had outside of this PR?